ccd-definition/CaseEvent/Camunda/BusinessProcessEvents.json

```diff
@@ -0,0 +1,47 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "DISPATCH_BUSINESS_PROCESS",
+    "Name": "Dispatch business process",
+    "Description": "Set READY business process status to DISPATCHED",
+    "DisplayOrder": 22,
```

Comment:
Should we remove `DisplayOrder` for any event that's not user event?

Reply:
Agree on the above, a big benefit of this is that we can accurately keep track of where events will be displayed for the user. A further improvement could be to split user events by state, but not a part of this :)

Reply:
Sounds fine as it is irrelevant. It's not part of this tech debt. but I can take care of that as well.

Reply:
@sabahirfan I would ignore state for now, as there is a big state refactor coming :)